examples/manipulating_visualizing/plot_spms.py

```diff
@@ -0,0 +1,61 @@
+"""
+Massively univariate analysis of a motor task from the Localizer dataset
+========================================================================
+
+This example shows the results obtained in a massively univariate
+analysis performed at the inter-subject level. 
+
+"""
+# Author: Bertrand Thirion, <bertrand.thirion@inria.fr>, Dec. 2015
+print __doc__
+
+import numpy as np
+import matplotlib.pyplot as plt
+from nilearn import datasets
+from nilearn.input_data import NiftiMasker
+
+### Load Localizer contrast ###################################################
+n_subjects = 20
+localizer_dataset = datasets.fetch_localizer_contrasts(
+    ['left vs right button press'], get_anats=True, n_subjects=n_subjects)
+# print basic information on the dataset
+print('First contrast nifti image (3D) is located at: %s' %
+      localizer_dataset.cmaps[0])
+contrast_map_filenames = localizer_dataset.cmaps
+anat = localizer_dataset.anats[0]  # for display
+
+### Mask data #################################################################
```

Comment:
Could you use the new notebook-style type of splitting: first a long line of "#" and then, right below, the comment. This will ensure that the example is well rendered by sphinx gallery.